# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/nsIWorkerDebugger.idl
# Commit: 1e774bd7f365
# Full Hash: 1e774bd7f36525a03b6260b1cd7741bcf4e5f622
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-04-09 04:12:49
# Regressor Bug: 1899503
# File Overlap Count: 4
# Description:
#   Bug 1899503 - P3 PRemoteWorkerDebugger binding implementation. r=asuth
#   
#   This patch implements the PRemoteWorkerDebugger binding.
#   
#   We are trying to keep the same logic debugger registration/unregistration to minimize the difference between the local debugger(the debugger on the content process main thread)
# ==============================================================================

diff -r b003e26209cf -r 1e774bd7f365 dom/workers/nsIWorkerDebugger.idl
--- a/dom/workers/nsIWorkerDebugger.idl	Tue Apr 08 09:43:33 2025 +0000
+++ b/dom/workers/nsIWorkerDebugger.idl	Tue Apr 08 09:43:33 2025 +0000
@@ -29,6 +29,8 @@
 
   readonly attribute boolean isChrome;
 
+  readonly attribute boolean isRemote;
+
   readonly attribute boolean isInitialized;
 
   readonly attribute nsIWorkerDebugger parent;